### PR TITLE
Fixes Issue#3493: Created getStandardMargin() in WidgetViewUtils to fix Margins 

### LINKED ISSUE #3493: Margin is not noticeable for widgets with buttons when a value is added manually
#### Software and hardware versions 
Collect v1.25

#### Problem description

Margin is NOT visible for widgets:
* Ex string when a value is added manually
* Ex integer when a value is added manually
* Ex decimal when a value is added manually
* Bearing when a value is added manually

#### Steps to reproduce the problem
1. User opens one on mentioned widgets
2. User clicks on the button
3. Text field appears
4. Margin is not visible for the text field

#### Expected behavior
Margin is noticeable 

#### Other information 
Noticed during testing #3464

### PR DESCRIPTION
**Pull request by SaumiaSinghal**
Closes #3493 

#### What has been done to verify that this works as intended?
I tested it on Android 9.0.

Before adding margin           |     After adding margin
:-------------------------:|:-------------------------:
Screenshot_20191230_034615  |  Screenshot_20191230_033317

#### Why is this the best possible solution? Were any other approaches considered?
I used the default margin value, that is being used in the other widgets, for uniformity. It adds the margin to the answer layout, where the answer field will appear. 

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
I don't think there are any regression risks.

#### Do we need any specific form for testing your changes? If so, please attach one.
Allwidges form

#### Does this change require updates to documentation? If so, please file an issue [here]( https://github.com/opendatakit/docs/issues/new) and include the link below.
No

#### Before submitting this PR, please make sure you have:
- [x] run `./gradlew checkAll` and confirmed all checks still pass OR confirm CircleCI build passes and run `./gradlew connectedDebugAndroidTest` locally.
- [x] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/opendatakit/collect/blob/master/collect_app/src/main/assets/open_source_licenses.html).
- [x] verified that any new UI elements use theme colors. [UI Components Style guidelines](https://github.com/opendatakit/collect/blob/master/CONTRIBUTING.md#ui-components-style-guidelines)

### CODE REVIEW

File: collect_app/src/main/java/org/odk/collect/android/formentry/questions/WidgetViewUtils.java [hunk 1]
```diff
@@ -19,13 +20,22 @@
...
 
     private WidgetViewUtils() {
 
     }
 
+    public static int getStandardMargin(Context context) {
+        Resources resources = context.getResources();
+        int marginStandard = dpFromPx(context, resources.getDimensionPixelSize(R.dimen.margin_standard));
+
+        return marginStandard - FIELD_HORIZONTAL_MARGIN_MODIFIER;
```
Comment (seadowg) [inline collect_app/src/main/java/org/odk/collect/android/formentry/questions/WidgetViewUtils.java]: Would be good to move this contstant to this file and name it something like `WIDGET_ANSWER_STANDARD_MARGIN_MODIFIER`.

**Review by seadowg — CHANGES_REQUESTED** [review]
This mostly looks good to me. I've left one comment for something I'd like changed. Also, it looks like adding margins to the Grid Widget hasn't quite worked when it's in a minimal appearance:

Screenshot_1581341179

Found this in the `AllWidgetsForm.xml` which is nice for testing this stuff out!

**Comment by SaumiaSinghal** [comment]
Hie @seadowg! Thanks for reviewing! I did the mentioned changes and also I removed the margin from grid widgets. It mostly looks good to me now than before.

**Comment by seadowg** [comment]
@SaumiaSinghal looks like there is still some PMD issues. If you fix them and make sure to run `./gradlew checkAll` I think we're all good!

**Comment by SaumiaSinghal** [comment]
Yes done! Sorry @seadowg. :sweat_smile:

**Review by seadowg — APPROVED** [review]

**Comment by mmarciniak90** [comment]
**Tested with success**

Verified on Android: 4.2, 4.4, 5.1, 6.0, 7.0, 8.1, 9.0, 10.0

**Verified cases:**
- all widgets from `All widgets` form
   - margin is visible on widgets with button(s), range widget, likert widget
- all widgets from `Birds` form
- RTL language
- light/dark mode
- horizontal/vertical device orientation

@opendatakit-bot unlabel "needs testing"
@opendatakit-bot label "behavior verified"

**Pull request merged**